ui/js/components/perf/compare.js

```diff
@@ -77,9 +77,9 @@ treeherder.component('phCompareTable', {
                     ctrl.filteredResultList[key] = compareResults;
                 }
             });
-            ctrl.filteredResultList = _.map(Object.keys(ctrl.filteredResultList), function (testName) {
-                return { testName: testName, results: ctrl.filteredResultList[testName] };
-            });
+            ctrl.filteredResultList = Object.keys(ctrl.filteredResultList).map(
```

Comment:
We could replace ``Object.keys`` here with ``Object.entries`` but since it's pre-existing, we don't need to worry about it.